packages/vhd-lib/src/Vhd/VhdDirectory.js

```diff
@@ -39,8 +39,8 @@ export class VhdDirectory extends VhdAbstract {
     this.#uncheckedBlockTable = blockTable
   }
 
-  static async open(handler, path) {
-    const vhd = new VhdDirectory(handler, path)
+  static async open(handler, path, opts) {
+    const vhd = new VhdDirectory(handler, path, { flags: 'r+', ...opts })
```

Comment:
I don't like these flags, they are too level for me to be exposed on `@xen-orchestra/fs` or here, but I don't have a much better idea at the moment.

At least, let's not allow passing other options:
```suggestion
  static async open(handler, path, { flags = 'r+' } = {}) {
    const vhd = new VhdDirectory(handler, path, { flags })
```